Assets/Scripts/UI/SwitchSelectMsg.cs

```diff
@@ -8,12 +8,24 @@ public class SwitchSelectMsg : MonoBehaviour {
 
 	private Controller2 controller2;
     public TextBoxManager textBoxManager;
-    // Use this for initialization
+
+	private checkOneTimeTrigger switchTriggeredCheck;
+	private checkOneTimeTrigger acceptTriggeredCheck;
+
+	private Renderer switchMessageRenderer;
+	private Renderer acceptMessageRenderer;
+
     //this is a tutotial script for level 1a
     void Start()
     {
         controller2 = GetComponent<Controller2>();
 		textBoxManager = GameObject.FindObjectOfType<TextBoxManager> ();
+
+		switchTriggeredCheck = switchMessage.GetComponent<checkOneTimeTrigger> ();
+		acceptTriggeredCheck = acceptMessage.GetComponent<checkOneTimeTrigger> ();
+		switchMessageRenderer = switchMessage.GetComponent<Renderer> ();
+		acceptMessageRenderer = acceptMessage.GetComponent<Renderer> ();
+
```

Comment:
Just a question, since I never really used renderer. Is this method better, and what does this exactly do? Annd should I use this type of method from now on?